DFC.ServiceTaxonomy.PageLocation/Validators/PageLocationsTaxonomyValidator.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.Content;
+using DFC.ServiceTaxonomy.Content.Services.Interface;
+using DFC.ServiceTaxonomy.Taxonomies.Helper;
+using DFC.ServiceTaxonomy.Taxonomies.Models;
+using DFC.ServiceTaxonomy.Taxonomies.Validation;
+using Newtonsoft.Json.Linq;
+using OrchardCore.ContentManagement;
+
+namespace DFC.ServiceTaxonomy.PageLocation.Validators
+{
+    public class PageLocationsTaxonomyValidator : ITaxonomyValidator
+    {
+        private readonly IContentItemsService _contentItemsService;
+        private readonly ITaxonomyHelper _taxonomyHelper;
+        private IEnumerable<ITaxonomyTermValidator> _validators;
+
+        public PageLocationsTaxonomyValidator(IContentItemsService contentItemsService, ITaxonomyHelper taxonomyHelper, IEnumerable<ITaxonomyTermValidator> validators)
+        {
+            _contentItemsService = contentItemsService;
+            _taxonomyHelper = taxonomyHelper;
+            _validators = validators;
+        }
+
+        public async Task<TaxonomyValidationResult> Validate(TaxonomyPart part)
+        {
+            List<string> errors = new List<string>();
```

Comment:
Think this should pass validation if it's not the page location taxonomy.
(Or taxonomy validators could declare the taxonomies they should be applied to.)

Reply:
Good spot, sorted!